recipes/thrift-cpp/build.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/env bash
+
+BOOST_ROOT=$PREFIX
+ZLIB_ROOT=$PREFIX
+LIBEVENT_ROOT=$PREFIX
+
+# If we build in local dev mode, use the bundled OpenSSL
+if [[ "$PRODUCTION" -eq "0" || "$OSTYPE" == "darwin"* ]]; then
+  OPENSSL_ROOT=$PREFIX
+  OPENSSL_ARGS=--with-openssl=$OPENSSL_ROOT
+else
+  OPENSSL_ARGS=
+fi
+
+if [ -d "${PIC_LIB_PATH:-}" ]; then
+  PIC_LIB_OPTIONS="--with-zlib=${PIC_LIB_PATH} "
+
+fi
+
+if [ "$(uname)" == "Darwin" ]; then
+  # C++11 finagling for Mac OSX
+  export CC=clang
+  export CXX=clang++
+  export MACOSX_VERSION_MIN="10.7"
+  export CXXFLAGS="${CXXFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export CXXFLAGS="${CXXFLAGS} -stdlib=libc++ -std=c++11"
+  export LDFLAGS="${LDFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export LDFLAGS="${LDFLAGS} -stdlib=libc++ -std=c++11"
+  export LINKFLAGS="${LDFLAGS}"
+
+  aclocal -I ./aclocal
+  libtoolize --copy
+  autoconf
+fi
+
+./configure --with-pic --prefix=$PREFIX \
+  --with-c_glib=no \
+  --with-php=no --with-java=no --with-perl=no --with-erlang=no --with-csharp=no \
+  --with-ruby=no --with-haskell=no --with-erlang=no --with-d=no \
+  --with-python=no \
+  --with-boost=${BOOST_ROOT} \
+  --with-zlib=${ZLIB_ROOT} \
+  --with-libevent=${LIBEVENT_ROOT} \
+  --with-nodejs=no \
+  --with-lua=no \
+  --with-go=no --with-qt4=no --with-libevent=no ${PIC_LIB_OPTIONS:-} $OPENSSL_ARGS
```

Comment:
:+1: Good to know :-)

@wesm, please add bash continuation lines (`\`) between all configure options. This makes way easier to read it. These are my 2 cents for good practices @jakirkham :-)